.gitlab-ci.yml

```
@@ -715,6 +715,7 @@ library:ci-math_classes:
 
 library:ci-mathcomp:
   extends: .ci-template-flambda
+  only: *full-ci
```

if you want to enable the job on light ci I think this works
```suggestion
  variables:
    FULL_CI: "true"
```